_data/projects/Artistic-Neural-Style-Transfer.yml

```diff
@@ -0,0 +1,11 @@
+name: Artistic Neural Style Transfer
+desc: Artistic Neural Style Transfer helps to transfer the style of your favourite artist style image to any content image.
+site: https://github.com/riya-17/Neural-Artistic-Style-Transfer
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- Tensorflow
+- python
+- numpy
```

Comment:
@riya-17 You can add more tags like Neural network, Scipy, Machine learning and many more.